core/src/com/po/fuck/view/Renderer.java

```diff
@@ -24,8 +26,9 @@
  * Class responsible for rendering game objects.
  */
 public class Renderer {
-    static private Map<Class<?>, ObjectDrawer<?> > drawers = new HashMap<>();
-    
+    public static final int MAX_Z = 20;
```

Comment:
Are we limiting the number of layers by `MAX_Z`? Probably we could get rid of this and rewrite other parts using `HashMap`?

Reply:
What are the other parts ?
If you are talking about `getLayers` then we can use `TreeMap` because we need to preserve layer order.

Reply:
Yes, about `getLayers`